### PR TITLE
Create watch model and front-end

### PR DESCRIPTION
**Pull request by jluria**
*Your PR Notes Here*

connect to #42

- [ ] Create watch model
- [ ] Add api endpoint for getting all watches
- [ ] Import watch model in model index
- [ ] Create an all watches component for viewing all watches
- [ ] Add front-end routing and import logic for rendering the all watches component

### CODE REVIEW

File: client/components/AllWatches.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux'
+
+const Watches = (props) => {
+  const { watches } = props;
+  return (
+    <div>
+      <ul>
```
Comment (schai6) [inline client/components/AllWatches.js]: move `<ul>` tags inside the map function.

File: client/components/AllWatches.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  )
+}
+
+const mapStateToProps = (state) => {
+  return {
+    watches: state.watches
+  }
+}
+
+export const AllWatchesContainer = connect(mapStateToProps)(Watches);
```
Comment (schai6) [inline client/components/AllWatches.js]: export default connect(mapStateToProps)(Watches)

see lessons.js

File: client/components/index.js [hunk 3]
```diff
@@ -6,5 +6,6 @@
 export {default as Main} from './main'
 export {default as UserHome} from './user-home'
 export {Login, Signup} from './AuthForm'
+export {AllWatchesContainer as AllWatches} from './AllWatches'
```
Comment (schai6) [inline client/components/index.js]: change to export {default as AllWatches} from './AllWatches. See previous comment.

File: server/db/models/watch.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  price: {
+    type: Sequelize.INTEGER,
+    allowNull: false
+  }
+})
+
+module.exports = Watch
+
+/**
```
Comment (schai6) [inline server/db/models/watch.js]: Remove comments

File: client/store/watches.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * ACTION CREATORS
+ */
+const getWatches = watches => ({type: GET_WATCHES, watches})
+
+/**
+ * THUNK CREATORS
+ */
+export const fetchWatches = () =>
```
Comment (schai6) [inline client/store/watches.js:16]: Need to add this to `fetchInitialData` in routes.js

**Review by schai6 — CHANGES_REQUESTED** [review]
Nice! Are you going to remove Lessons/Lesson completely before merging? Still need a single Watch view.

File: client/store/index.js [hunk 6]
```diff
@@ -3,10 +3,11 @@ import createLogger from 'redux-logger'
 import thunkMiddleware from 'redux-thunk'
 import { composeWithDevTools } from 'redux-devtools-extension'
 import user from './user'
-import lessons from './lessons'
+import lessons from './lessons';
```
Comment (johnson-alan) [inline client/store/index.js]: semi-colon

File: client/store/index.js [hunk 7]
```diff
@@ -3,10 +3,11 @@ import createLogger from 'redux-logger'
...
 import lesson from './lesson'
+import watches from './watches';
```
Comment (johnson-alan) [inline client/store/index.js]: semi-colons

**Review by jemyishai — APPROVED** [review]

**Review by schai6 — APPROVED** [review]

**Pull request merged**